.github/workflows/programequity_slack.yml

```
@@ -0,0 +1,24 @@
+on:
+  issues:
+    types: [opened, edited]
+  issue_comment:
+    types: [created, edited]
+  pull_request:
+    types: [opened, edited, review_requested]
+  pull_request_review:
+    types: [submitted]
+  pull_request_review_comment:
+    types: [created, edited]
+
+jobs:
```

I also prefer to set explicit permissions requirements:

```suggestion
permissions:
  issues: read
  pull-requests: read

jobs:
```